practice_app/api/urls.py

```diff
@@ -1,16 +1,19 @@
 from django.urls import path
 from . import views
 
+
+
 app_name = "api"
 urlpatterns = [
     path("doaj-api/", views.doaj_get, name="doaj_api"),
     path("google-scholar/", views.google_scholar, name="google-scholar"),
-    path("core", views.core_get, name="core"),
+    path("core/", views.core_get, name="core"),
```

Comment:
Just a quick suggestion for future changes: I believe, separating unrelated changes into separate pull requests is better practice